src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -695,6 +695,13 @@ def transform_metrics_list_output(result):
     return new_result
 
 
+def create_boolean_result_output_transformer(property_name):
+    def _transformer(result):
+        return {property_name: result}
+
+    return _transformer
+
+
 def transform_storage_boolean_output(result):
```

Comment:
Looks like most of them have been switched anyway.
Then can delete this method.